src/schema/schema.graphql

```diff
@@ -2200,6 +2213,7 @@ type Query {
 	getSMimePublicCerts(contactAddr: String!, store: String!): SMimePublicCertsResponse
 	getSearchFolder: Folder
 	getWhiteBlackList: WhiteBlackList
+	getWorkingHours(names: [String!], start: Float, end: Float): [WorkingHours]
```

Comment:
shouldn't the name also be a required parameter? if someone executes this query with names as null, it might fail in the the function which prepares the jsonRequest as we are doing `names.join` there.

Reply:
done